### ISSUE TITLE
java.lang.AbstractMethodError on org.eclipse.aether.connector.basic.BasicRepositoryConnector.get()

### ISSUE THREAD

**Issue by jruaux** [kordamp/pomchecker#6]
Getting this error when running pomchecker against the attached pom

```
/pomchecker-1.5.0/bin/pomchecker check-maven-central --file=riot-db-2.18.6-20230121.010503-1.pom
[INFO] Maven Central checks: /Users/jruaux/git/riot/build/repos/local/release/com/redis/riot-db/2.18.6-SNAPSHOT/riot-db-2.18.6-20230121.010503-1.pom
Exception in thread "main" java.lang.AbstractMethodError: Receiver class org.eclipse.aether.internal.impl.Maven2RepositoryLayoutFactory$Maven2RepositoryLayoutEx does not define or inherit an implementation of the resolved method 'abstract java.util.List getChecksumAlgorithmFactories()' of interface org.eclipse.aether.spi.connector.layout.RepositoryLayout.
	at org.eclipse.aether.connector.basic.BasicRepositoryConnector.get(BasicRepositoryConnector.java:220)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.performDownloads(DefaultArtifactResolver.java:514)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:402)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifacts(DefaultArtifactResolver.java:229)
	at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifact(DefaultArtifactResolver.java:207)
	at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveArtifact(DefaultRepositorySystem.java:262)
	at org.apache.maven.project.ProjectModelResolver.resolveModel(ProjectModelResolver.java:192)
	at org.apache.maven.model.building.DefaultModelBuilder.importDependencyManagement(DefaultModelBuilder.java:1347)
	at org.apache.maven.model.building.DefaultModelBuilder.build(DefaultModelBuilder.java:544)
	at org.apache.maven.model.building.DefaultModelBuilder.build(DefaultModelBuilder.java:454)
	at org.apache.maven.model.building.DefaultModelBuilder.build(DefaultModelBuilder.java:267)
	at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:173)
	at org.apache.maven.project.DefaultProjectBuilder.build(DefaultProjectBuilder.java:124)
	at org.kordamp.maven.checker.cli.internal.PomParser.createMavenProject(PomParser.java:99)
	at org.kordamp.maven.checker.cli.internal.PomParser.createMavenProject(PomParser.java:66)
	at org.kordamp.maven.checker.cli.CheckMavenCentral.execute(CheckMavenCentral.java:45)
	at org.kordamp.maven.checker.cli.AbstractCommand.call(AbstractCommand.java:99)
	at org.kordamp.maven.checker.cli.CheckMavenCentral.call(CheckMavenCentral.java:30)
	at org.kordamp.maven.checker.cli.AbstractCommand.call(AbstractCommand.java:33)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
	at picocli.CommandLine.execute(CommandLine.java:2170)
	at org.kordamp.maven.checker.cli.Main.execute(Main.java:69)
	at org.kordamp.maven.checker.cli.Main.run(Main.java:55)
	at org.kordamp.maven.checker.cli.Main.main(Main.java:47)
```

[riot-db-2.18.6-20230121.010503-1.pom.txt](https://github.com/kordamp/pomchecker/files/10471235/riot-db-2.18.6-20230121.010503-1.pom.txt)

**Comment by aalmiray** [comment]
Released in v1.6.0 -> https://github.com/kordamp/pomchecker/releases/tag/v1.6.0

**Comment by jruaux** [comment]
Unfortunately the error still happens with maven-resolver 1.9.4. I believe this is due to maven-core using maven-resolver 1.6.3.
